### PR TITLE
Updated 10MinuteMail description

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
**Added some valuable piece of information to the description of 10MinuteMail.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [MPCrypt](http://crypt.miguelpiedrafita.com/) - Securely encrypt files.
 * [jotti](https://virusscan.jotti.org) - Scan any suspicious file or email attachment for viruses.
 * [scr.im](http://scr.im/) - Share e-mail address freely without worrying about spam.
-* [10 Minute Mail](https://10minutemail.com/) - A temporary fully functional email address for 10 minutes.
+* [10 Minute Mail](https://10minutemail.com/) - A temporary fully functional email address for 10 minutes(extendible).
```
Comment (aviaryan) [inline README.md]: Add space between `(` and 'minutes'.

Comment (Anshuman-Verma) [inline README.md]: Fixed

**Comment by Anshuman-Verma** [comment]
@aviaryan we have multiple instances of no space between '(' and the text before, should i add spacing in all too ?

**Comment by aviaryan** [comment]
@Anshuman-Verma Sure.

**Pull request merged**